### ISSUE TITLE
Polar calculation and description of arcs

### ISSUE THREAD

**Issue by mxfh** [parametric-svg/spec#5]
Consider expanding the spec to parse trigonometric functions to enable polar coordinate description of arcs and other objects. 

May be even a shorthand parsing center radius and angle and angleOffset.

http://stackoverflow.com/questions/5736398/how-to-calculate-the-svg-path-for-an-arc-of-a-circle

**Comment by tomek-he-him** [comment]
Yup, trigonometric functions are definitely on the roadmap. They’ll be landing in the spec and https://github.com/parametric-svg/parse over the next weeks.

**Comment by tomek-he-him** [comment]
> May be even a shorthand parsing center radius and angle and angleOffset.

@mxfh, I’d rather keep this spec low-level. Keep lightweight and avoid [scope creep](https://en.wikipedia.org/wiki/Scope_creep). I only want to build in fundamental functions which are impossible to build out of other functions and operators.

Feel free to suggest features at #10!

**Comment by tomek-he-him** [comment]
Hi @mxfh! I’m working on built-in functions here: https://github.com/parametric-svg/-/tree/built-in-functions-%2310/packages/parametric-svg-spec#-built-in-functions.

I’m going to merge these into master tomorrow (about 17 hours from now) – feel free to suggest more features if you need them!

**Comment by mxfh** [comment]
Great news, was a bit dormant on this but I'm sure that I'll actually try out some stuff in the next days.

I tried to hack animations for parametric behaviour within the SVG specs before, but by now this seems like an unmaintable excercise, and with SMIL animations becoming deprecated this approach just becomes impractical if it requires a polyfill. 
http://codepen.io/mxfh/pen/yYMOZP?editors=101

**Comment by tomek-he-him** [comment]
Hi @mxfh! I must admit there were a couple of rough edges at the [built-in-functions-#10](https://github.com/parametric-svg/-/tree/built-in-functions-%2310) branch – and my resources for that hack streak ran out. So it still remains unmerged.

But – guess what – I was planning to get back to this today anyway! So stay tuned!

😃
